tests/components/rpi_power/test_config_flow.py

```diff
@@ -0,0 +1,42 @@
+"""Tests for rpi_power config flow."""
+from homeassistant.components.rpi_power.const import DOMAIN
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+from tests.async_mock import MagicMock
+from tests.common import patch
+
+MODULE = "homeassistant.components.rpi_power.config_flow.new_under_voltage"
+
+
+async def test_setup(hass: HomeAssistant):
+    """Test setting up manually."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": SOURCE_USER},
+    )
+    assert result["type"] == RESULT_TYPE_FORM
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)